xrpl/ledger/main.py

```diff
@@ -2,11 +2,12 @@
 
 import asyncio
 
+# from xrpl.clients.sync_client import SyncClient
+import xrpl.clients.sync_client as sync_client_ref
```

Comment:
Why is this not just imported as `sync_client`? IMO `sync_client_ref` is more confusing

Reply:
Importing as before!